src/dotnet-interactive/CommandLine/CommandLineParser.cs

```diff
@@ -171,9 +171,16 @@ Command Jupyter()
                 description: LocalizationResources.Cli_dotnet_interactive_jupyter_install_http_port_range_Description(),
                 isDefault: true);
 
+            var localOnlyNetworkInterfacesOption = new Option<bool>(
+                "--local-only-network-interfaces",
+                description: LocalizationResources.Cli_dotnet_interactive_jupyter_local_only_network_interfaces_Description(),
+                getDefaultValue: () => false
```

Comment:
There's no need to specify a default value since `false` is already the default.